Cargo.toml

```diff
@@ -18,17 +19,19 @@ byteorder = "1.4.3"
 bytes = "1.1.0"
 enumn = "0.1.4"
 libva = { git = "https://github.com/chromeos/cros-libva", rev = "843cef6", package = "cros-libva", optional = true }
+v4l2r = { git = "https://github.com/bgrzesik/v4l2r", rev = "59a2526", package = "v4l2r", optional = true }
```

Comment:
I'll replace with `https://github.com/Gnurou/v4l2r` once https://github.com/Gnurou/v4l2r/pull/35 is merged